File: java/program/Clothing.java
class Clothing extends Product {
    protected String size;
    protected String material;
    protected String gender;

    public Clothing(String idProduct, String name, String brand, double price, String size, String material, String gender) {
        super(idProduct, name, brand, price);
        this.size = size;
        this.material = material;
        this.gender = gender;
    }

    public String getSize() {
        return size;
    }

    public String getMaterial() {
        return material;
    }

    public String getGender() {
        return gender;
    }

    public void setSize(String size) {
        this.size = size;
    }

    public void setMaterial(String material) {
        this.material = material;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }
}

File: python/program/Clothing.py
from Product import Product

class Clothing(Product):
    def __init__(self, id_product, name, brand, price, size, material, gender):
        super().__init__(id_product, name, brand, price)
        self.size = size
        self.material = material
        self.gender = gender

    def get_size(self):
        return self.size

    def get_material(self):
        return self.material

    def get_gender(self):
        return self.gender

    def set_size(self, size):
        self.size = size

    def set_material(self, material):
        self.material = material

    def set_gender(self, gender):
        self.gender = gender